lib/packwerk/inflector.rb

```diff
@@ -32,8 +49,6 @@ def pluralize(word, count = nil)
       end
     end
 
-    private
```

Comment:
Removed this as a private method because the `ApplicationValidator` needs to [check the inflections ](https://github.com/Shopify/packwerk/pull/53/files#diff-5ca9dcd54fd0b9eeef94e00ba6adc7822dc83e5dd2bfb9e8d96fd3c356de8b8aR179)of this class.